src/controllers/IllnessStartDateController.ts

```diff
@@ -0,0 +1,75 @@
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { OK, UNPROCESSABLE_ENTITY } from 'http-status-codes';
+import { controller, httpGet, httpPost } from 'inversify-express-utils';
+
+import { BaseAsyncHttpController } from 'app/controllers/BaseAsyncHttpController';
+import { FormValidator } from 'app/controllers/validators/FormValidator';
+import { IllnessReasonFeatureMiddleware } from 'app/middleware/IllnessReasonFeatureMiddleware';
+import { schema } from 'app/models/fields/IllnessStartDate.schema';
+import { ILLNESS_START_DATE_PAGE_URI } from 'app/utils/Paths';
+import { ValidationResult } from 'app/utils/validation/ValidationResult';
+
+const template = 'illness/illness-start-date';
+const ILLNESS_START_DAY_FIELD = 'startDay';
+const ILLNESS_START_MONTH_FIELD = 'startMonth';
+const ILLNESS_START_YEAR_FIELD = 'startYear';
+const ILLNESS_FULL_DATE = 'startDate';
```

Comment:
why are these capitalised?

Reply:
i'm used to constants being capitalised especially when their value does not change. But i'm happy to make these lowercase to be consistent with the project